test/unit/helpers/meta/check-donator-status.test.js

```diff
@@ -0,0 +1,14 @@
+const { expect } = require('chai');
+
+describe('HELPER donator status', () => {
+
+  it('Caches the response', async () => {
+    const spy = sandbox.stub(sails.helpers.meta, 'getDonatorStatus').resolves('sponsor');
```

Comment:
Could use a better test here to see if the cache actually expires but since it gets sent to Redis that's hard to test. There's no time-bending possible there.

But I think it's safe to trust that Redis will properly expire the keys as needed :)